test/org/sagebionetworks/bridge/services/StormPathUserAdminServiceTest.java

```diff
@@ -150,4 +163,31 @@ public void testDeleteUserThatHasBeenDeleted() {
         userAdminService.deleteUser(study, session.getUser().getEmail());
         assertNull(authService.getSession(session.getSessionToken()));
     }
+    
+    @Test
+    public void creatingUserConsentsToAllRequiredConsents() {
+        Study study = null;
+        try {
+            study = TestUtils.getValidStudy(StormPathUserAdminServiceTest.class);
+            studyService.createStudy(study);
```

Comment:
Creating a study is a very heavy-weight operation. Is there any way we can run this test without creating and destroying a study every time?